assets/sass/app.scss

```diff
@@ -10,3 +10,31 @@
 .page-header {
   margin-bottom: $govuk-gutter;
 }
+
+.expanding-panel {
+  border-top: none;
+  &:last-child {
+    border-bottom: none;
+  }
+
+  .content {
+    padding: $govuk-gutter-half;
+    padding-left: 0;
+  }
+}
+
+.govuk-heading-m, .application-confirm legend, .panel-list li {
+  font-size: inherit;
+  font-weight: inherit;
+  line-height: inherit;
+}
+
+.separator {
+  border-bottom: 1px solid $govuk-border-colour;
```

Comment:
cant this be achieved with `<hr />`?

Reply:
actually, it's not in the HTML anymore so lets remove it

Reply:
done